Makefile

```diff
@@ -64,6 +64,16 @@ trusted_os: APP=trusted_os
 trusted_os: DIR=$(CURDIR)/trusted_os
 trusted_os: create_dummy_applet elf
 
+trusted_os_signed: trusted_os
```

Comment:
Who is using this target (instead of `trusted_os_embedded_applet`)?

Reply:
me :)

`trusted_os_embedded_applet` is good for testing via SDP or on `QEMU`, but we need this to be able to end-to-end test on device with the full bootloader->os->applet loader chain.